src/apis/reviewsApi.ts

```
@@ -11,3 +37,69 @@ export const getReviews = async (reviewId: string): Promise<Review> => {
+      const response = await axiosAuthClient.post<Review>("/reviews", preparedData);
+      console.log("리뷰 작성 성공 (방법 1):", response.status);
+      return response.data;
+    } catch (err: any) {
+      throw err;
+    }
+  } catch (error: any) {
+    console.error("리뷰 남기기 API 오류:", error.message);
+
+    if (error.response) {
+      console.error("상태 코드:", error.response.status);
+      console.error("응답 데이터:", error.response.data);
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)